lib/saml_idp/fingerprinting.rb

```diff
@@ -0,0 +1,7 @@
+module SamlIdp
+  module Fingerprinting
+    def self.from_cert(cert)
+      Digest::SHA256.hexdigest(OpenSSL::X509::Certificate.new(cert).to_der).scan(/../).join(':')
```

Comment:
I was asked if we can use `SHA512` here... should this depend on configuration? 
if my `config.algorithm` is set to `:sha512`, I need to use SHA512 for the fingerprint, digest, is that correct?

Reply:
I suppose I would think so yes - could make the algorithm a param here for fingerprint so it's up to the caller?